### DISCUSSION TITLE
A fun idea

### DISCUSSION THREAD

**Discussion by Deadtrix21** [nextcord/nextcord#338] [Q&A] [answered]
Add a way to grab global errors
Not just command errors

But processing errors too
It's something that could be widely used

**Comment by VincentRPS** [comment, marked as answer]
on_error does iirc

**Comment by Deadtrix21** [comment]
> on_error does iirc

Wait really???

**Comment by ooliver1** [comment]
yes it does, see the docs if you're unsure
